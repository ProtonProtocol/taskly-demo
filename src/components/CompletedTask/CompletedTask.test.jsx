import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import CompletedTask from './CompletedTask';

const createTestProps = props => ({
  text: 'test',
  ...props,
});

describe('CompletedTask component', () => {
  let props;

  beforeEach(() => {
    props = createTestProps({});
  });

  it('renders without crashing', () => {
    shallow(<CompletedTask {...props} />);
  });

  it('matches the rendered snapshot', () => {
    const component = shallow(<CompletedTask {...props} />);
    expect(toJson(component)).toMatchSnapshot();
  });

});
