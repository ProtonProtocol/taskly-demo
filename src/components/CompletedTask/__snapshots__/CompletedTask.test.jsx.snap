// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CompletedTask component matches the rendered snapshot 1`] = `
<div
  className="completed-task"
>
  <CheckIcon />
  <span>
    test
  </span>
</div>
`;
